.github/workflows/release.yml

```
@@ -43,11 +43,15 @@ jobs:
         run: |
           rm -rf node_modules
           npm i node@v20-lts --no-save
+          npm install grunt-cli -g
+
+      - name: Grunt build and package
+        run: grunt default
```

You have all the dependencies/devDependencies needed to build your package defined in your package.json and NodeJS and npm is set up in the steps before. Why don't you use them?

```suggestion
     - name: Install dependencies & build package
       run: |
         npm install
         npm run build
```